src/cljs/main/broadfcui/common.cljs

```diff
@@ -210,3 +210,57 @@
                 :alert)
     [:span {:style {:fontWeight "bold" :fontSize "98%"}}
       "FireCloud is not intended to host personally identifiable information. Do not use any patient identifier, including name, social security number, or medical record number."]])
+
+(react/defc FoundationTooltip
+  {:component-did-mount
+   (fn [{:keys [this]}]
+     (.foundation (js/$ (react/find-dom-node this))))
+   :render
+   (fn [{:keys [props]}]
+     (let [{:keys [position text tooltip]} props]
+       [:span {:data-tooltip "" :className (str "has-tip " position)
+               :title tooltip}
+        text]))})
+
+(react/defc FoundationInfoBox
+  {:component-will-mount
+   (fn [{:keys [locals]}]
+     (swap! locals assoc :infobox-id (gensym "infobox-")))
+   :component-did-mount
+   (fn [{:keys [locals props]}]
+     (let [infobox-container (.createElement js/document "div")
+           infobox-id (:infobox-id @locals)]
+       (swap! locals assoc :infobox-container infobox-container)
+       (.insertBefore js/document.body infobox-container (.getElementById js/document "app"))
```

Comment:
This introduces a second location where we use this critical ID. Abstract it out.